config/locales/alchemy.en.yml

```diff
@@ -161,6 +161,7 @@ en:
         blank: "%{field} can't be blank"
         invalid: "%{field} has wrong format"
         taken: "%{field} has already been taken"
+        too_short: "%{field} is too short"
```

Comment:
This probably needs `too_long`, as well.

Reply:
this feature will be gone in #2984